editor/src/dev/load_project.clj

```diff
@@ -44,7 +45,7 @@
   ;; You are responsible for calling (task-fn) and returning its return value.
   (case task-key
     ;; We can selectively activate and deactivate profiling around these tasks.
-    (:make-nodes :read-resources :load-nodes)
+    (:read-resources :load-nodes)
```

Comment:
`:make-nodes` is no longer a separate step. It has been merged into `:load-nodes`.